### PR TITLE
[Operator Development] softplus

### LINKED ISSUE #882: Code Contribution: 【Easy】【Operator Development】softplus
## Description 任务介绍
Develop forward function for softplus operator.
开发softplus算子的前向功能。
## Requirements 任务要求
__Interface 接口__
`softplus(Tensor self, Scalar beta=1, Scalar threshold=20) -> Tensor`
__Function reference 功能参考__
https://docs.pytorch.org/docs/stable/generated/torch.nn.Softplus.html
__Implementation reference 实现参考__
https://github.com/FlagOpen/FlagGems/blob/master/src/flag_gems/ops/relu.py

The operator should support all optional arguments defined in the interface. 
算子应支持接口中定义的所有参数选项。
## DDL 提交时间
Please submit a Pull Request within 2 weeks after accepting the assignment.
请于接取任务后两周内提交PR。
Please provide both accuracy test and performance test code.
请同时提供实现正确性测试与性能测试代码。

### PR DESCRIPTION
**Pull request by 6eanut**
### PR Category
Operator , OP Test , Benchmark

### Type of Change
New Feature

### Description
Develop forward function for softplus operator.

### Issue
- Resolves #882 

### Progress

- [x] Change is properly reviewed (1 reviewer required, 2 recommended).
- [x] Change is responded to an issue.
- [ ] Change is fully covered by a UT.

### Performance

image
image
image

### CODE REVIEW

**Comment by meinie0826** [comment]
looks great to me !

**Review by meinie0826 — DISMISSED** [review]

**Review by StrongSpoon — APPROVED** [review]
lg
